src/Adapter/MSTest.CoreAdapter/Resources/Resource.resx

```diff
@@ -296,4 +296,7 @@
   <data name="UTA_CleanupStackTrace" xml:space="preserve">
     <value>TestCleanup Stack Trace</value>
   </data>
+  <data name="DiscoveryWarning" xml:space="preserve">
```

Comment:
Thanks Sushil. Could you run build with -uxlf so the corresponding xlf files are updated for localization.

Reply:
updated. pl check

Reply:
LGTM.